### PR TITLE
Add to modal

### PR DESCRIPTION
**Pull request by elizabethcan**
Added some styling and content to my modal. More work needs to be done here. Also made it so that you can click on the area outside the main modal to close it and removed the `Modal` button to open the modal, instead linking it to the image that is clicked on. Removed styling from the `<` `>` buttons in the photo carousel.

screen shot 2019-02-13 at 11 35 53 am

screen shot 2019-02-13 at 11 38 04 am

### CODE REVIEW

File: client/src/components/Container.jsx [hunk 1]
```diff
@@ -24,12 +24,11 @@ class Container extends React.Component {
...
-        <ReviewContainer />
-        <ReviewContainer />
-        <ReviewPhotosContainer />
-        <button onClick={this.toggleModal}>Modal</button>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewContainer showPrice="false"/>
+        <ReviewPhotosContainer openModal={this.toggleModal}/>
         <Modal showModal={this.state.showModal} onClose={this.toggleModal}/>
```
Comment (JonBeyond) [inline client/src/components/Container.jsx:32]: what does `onClose` do? it's not in the react docs and there's sparse info online

File: client/src/components/Modal.jsx [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
     return null;
   } else {
     return (
-      <div style={styles.modal}>
+      <div style={styles.modal} onClick={props.onClose}>
         <section style={styles.modalMain}>
-          <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
-          <button onClick={props.onClose}>Close</button>
+          {/* <div style={styles.reviewContent}> */}
+          <img style={styles.image} src="https://i.etsystatic.com/iap/c08960/1735567682/iap_640x640.1735567682_9rntz5h8.jpg?version=0"></img>
```
Comment (JonBeyond) [inline client/src/components/Modal.jsx:14]: I highly recommend you don't link to etsy's images.  save them locally and serve them from your static files/S3

File: client/src/components/Modal.jsx [hunk 3]
```diff
@@ -1,15 +1,24 @@
...
+          <ReviewContainer showPrice="true"/>
+          {/* </div> */}
+          {/* <div> */}
+          <button style={styles.button} onClick={props.onClose}>
+            <img style={styles.buttonImg}src="https://cdn0.iconfinder.com/data/icons/elasto-online-store/26/00-ELASTOFONT-STORE-READY_close-512.png"></img>
```
Comment (JonBeyond) [inline client/src/components/Modal.jsx:19]: likewise here, image leeching

File: client/src/components/ReviewPhotosCarousel.jsx [hunk 4]
```diff
@@ -4,11 +4,11 @@ import styles from './css/reviewPhotosCarousel.css.js';
 const ReviewPhotosCarousel = (props) => {
   return (
     <div>
-      <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0" style={styles.reviewImage}></img>
-      <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0" style={styles.reviewImage}></img>
-      <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0" style={styles.reviewImage}></img>
-      <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0" style={styles.reviewImage}></img>
-      <img src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0" style={styles.reviewImage}></img>
+      <img onClick={props.openModal} style={styles.reviewImage} src="https://i.etsystatic.com/iap/c08960/1735567682/iap_300x300.1735567682_9rntz5h8.jpg?version=0"></img>
```
Comment (JonBeyond) [inline client/src/components/ReviewPhotosCarousel.jsx:7]: (same.. image leeching)

**Review by JonBeyond — APPROVED** [review]
No major comments, just avoid image leeching as a general practice.  I know we are just locally developing here, but it's easy enough to serve files locally.

File: client/src/components/Modal.jsx (hunk 2 above)
Comment (elizabethcan) [reply client/src/components/Modal.jsx:14]: I started with hard coding some data to get some styling going. I starting to work on getting my backend hooked up now

**Pull request merged**